examples_test.go

```diff
@@ -67,3 +67,31 @@ func ExampleScan_parseDuration() {
 	// Output:
 	// 3m20s
 }
+
+func ExampleScan_repeatedly() {
+	line := []byte("www.google.com:1234 www.google.com:2345")
+	r := regexp.MustCompile(`(\S+):(\d+)`)
+
+	for {
+		var (
+			rng  re.Range
+			host string
+			port int
+		)
+		err := re.Scan(r, line, &rng, &host, &port)
+		if _, notFound := err.(re.NotFoundError); notFound {
```

Comment:
I think you can change this to:
   if errors.Is(re.NotFoundError) {  ... }

I think this API showed up in Go 1.13.